Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -7,16 +7,37 @@ namespace BootCamp.Chapter
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
-    public static class CaesarCipher
+    public class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return EncryptString(message, shift);
         }
 
+        private static string EncryptString(string message, int shift)
```

Comment:
Put private methods below public. Because private is a detail you might not care about. Public gives us a clear understanding of what we can do with the class, private gives a better understanding how does it work under the hood.